runtime/gc_stats/VLHGCIncrementStats.hpp

```diff
@@ -117,6 +117,14 @@ class MM_VLHGCIncrementStats : public MM_Base
 		_copyForwardStats.merge(&stats->_copyForwardStats);
 		_irrsStats.merge(&stats->_irrsStats);
 	}
+
+	/**
+	 * Get total stall time 
+	 */
+	uint64_t getTotalStallTime()
```

Comment:
I think we should rename this to `getIncrementStallTime`

Reply:
Both class and instance name has 'Increment' in them.  I tend to avoid to repeat those attributes in method names. 
I'd either keep getTotalStallTime or simply call it getStallTime (that's the name used in various OMR Stats struct).